build/DownloadedPackageVersions.targets

```diff
@@ -0,0 +1,11 @@
+<Project ToolsVersion="15.0">
+  <Target Name="DownloadPackageVersionsProps">
+    <PropertyGroup>
+      <PB_AssetRootAccessTokenSuffixParam Condition=" '$(PB_AssetRootAccessTokenSuffix)' != '' ">?$(PB_AssetRootAccessTokenSuffix)</PB_AssetRootAccessTokenSuffixParam>
+    </PropertyGroup>
+
+    <DownloadFile Uri="$(PB_PackageVersionPropsUrl)?$(PB_AssetRootAccessTokenSuffixParam)"
```

Comment:
If a token is not provided, your url will end up with a question mark at the end of it.

Reply:
Fixed...